### DISCUSSION TITLE
Command Line integration

### DISCUSSION THREAD

**Discussion by liam-maps** [StephaneCouturier/Katalog#510] [Ideas]
Hi!
I've been looking for so long for an app like this and I like it so much!
However, I'd be great to have CLI mode to run batch updates of the catalogs instead of managing through a GUI.
Have you considered it?
You looke talented enough to achieve it.
Thank you for your work.

**Comment by liam-maps** [comment]
Almost a year has passed @StephaneCouturier 
Is this project dead or what?
Maybe it's not a priority, or you don't know how to achieve it, but a little feedback would be appreciated.

**Reply by StephaneCouturier** [reply]
Hi,

This feature would be quite useful, thanks for submitting it.
Multiple report channels, I missed this one, and this is still a 1 person project.

It seems there are Qt functions to implement the command line interface and the good news is that they are cross-platform.
I will give it a try, here's the backlog entry:
https://github.com/StephaneCouturier/Katalog/issues/581

**Comment by StephaneCouturier** [comment]
last release also includes searching via cmd line
